tests/test_blocks.py

```diff
@@ -255,6 +255,62 @@ def test_code_block():
     assert markdown == "``` javascript\nconst a = 3\n```\n"
 
 
+def test_link_to_page_block():
+
+# As I've been asked to update the end-to-end tests, but not specifically the unit tests
+# changes to this file are out of scope.  Right? 
+
+
+    mock_page_id = "23143143243241423"
+    
+    # This test is not about representing the block itself, which is only a page link.
+    # It's about testing that a mock page, linked to by a mock block, is processed as expected.
+
+    # Elaboration: As the link_to_page block is trivial, containing only the value of the link itself.
+    # And since representation of the page link in the pandoc output is not necessary --
+    # It is the contents of the linked page that is potentially
+    # represented, not the link to the page -- testing here would logically be limited to checking
+    # for proper conversion of the linked page.  However, such a test would be redundant, 
+    # as already handled in test-end-to-end.py
+
+    # Create the mock block of the linking page.
+    notion_block = mock_block("link_to_page", { 
```

Comment:
Nice job identifying the need to use `mock_block` here, but you're not calling it correctly. See the code for how mock_block is implemented to see why:

https://github.com/innolitics/n2y/blob/main/n2y/notion_mocks.py#L94

In particular, you can just use `mock_block("link_to_page", {"type": "page_id", "page_id":  "{mock_page_id}"})`